_tests/integration/init_test.go

```diff
@@ -104,3 +106,81 @@ func Test_InitTest(t *testing.T) {
 		require.EqualError(t, err, "exit status 1", out)
 	}
 }
+
+func Test_GitignoreTest(t *testing.T) {
+	t.Log("create .gitignore with pattern when .gitignore does not exist")
```

Comment:
Can you add a test if .bitrise.secrets.yml is already included in the secrets file?